koku/api/metrics/serializers.py

```diff
@@ -14,41 +14,13 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
-"""CostModelMetricMap Model Serializer."""
-import logging
-
-from django.utils.translation import ugettext as _
+"""CostModelMetricMap Serializer."""
 from rest_framework import serializers
 
-from api.metrics.models import CostModelMetricsMap
-from api.models import Provider
-
-LOG = logging.getLogger(__name__)
-
-SOURCE_TYPE_MAP = {
-    Provider.PROVIDER_OCP: "OpenShift Container Platform",
-    Provider.PROVIDER_AWS: "Amazon Web Services",
-    Provider.PROVIDER_AZURE: "Microsoft Azure",
-}
-
-
-def error_obj(key, message):
-    """Create an error object."""
-    error = {key: [_(message)]}
-    return error
-
-
-class CostModelMetricMapSerializer(serializers.ModelSerializer):
-    """Serializer for the CostModelMetricsMap model."""
-
-    class Meta:
-        """Metadata for the serializer."""
 
-        model = CostModelMetricsMap
-        exclude = ("id",)
+class CostModelMetricMapSerializer(serializers.Serializer):
```

Comment:
What does this class do? If the `to_representation just hands back the data that's given it seems like this is just the `serializers.Serializer` class and we don't need it at all?

Reply:
Hi Chris, when I deleted the serializer, Django gave me an error that a serializer_class was needed in the ViewSet, or to override get_serializer_class(). When I overrided the get_serializer_class(), the data came back as empty. What do you suggest instead?

Reply:
Did you try this https://github.com/project-koku/koku/pull/1986#discussion_r402409433 and it didn't work? Or just not provide a serializer?

Reply:
nevermind, I'll plan to use the default django Serializer instead of this class

Reply:
@chambridge yes, I tried https://github.com/project-koku/koku/pull/1986#discussion_r402409433

Reply:
ok. I guess keep the existing serializer. I guess the `to_representation` method is what is making this work.